src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/DynamicPartGraphType.cs

```diff
@@ -34,9 +34,6 @@ public override void Initialize(ISchema schema)
             }
         }
 
-        // Part is not required here anymore, do not keep it alive.
-        _part = null;
```

Comment:
There is no need to keep this referenced. The schema is a singleton, which effectively keeps that reference alive forever.

Reply:
Where is it being registered as singleton? The `DynamicPartGraphType` is instantiated from `DynamicContentTypeBuilder`. I don't think it lives for a long time.

Reply:
The graph types are kept alive in the schema, IIRC.

Reply:
Not sure. SchemaService is a singleton and hold ISchema instances. But part is not something we keep in the Schema. I could be wrong, but please double check before we add it back.

Reply:
If this works in the c'tor it is much better. I don't remember why, but there was a reason having this in `Intialize`

Reply:
It works either way. I think with GraphQL 8 the expected is for it to be in the constructor. I could be wrong. Not very familiar with the library.

Reply:
Seems right, thanks for all of your work.